### PR TITLE
[5.2] Fluent json querying for postgres

### PR DESCRIPTION
**Pull request by themsaid**
Same as https://github.com/laravel/framework/pull/12685 but for Postgres.

``` php
User::select('name->en')
    ->where('contacts->phone->home', 123)
    ->orWhere('contacts->phone->mobile', 123)
    ->whereBetween('specs->height', [100, 120])
    ->orderBy('specs->weight')
    ->get();
```

### CODE REVIEW

File: src/Illuminate/Database/Query/Grammars/PostgresGrammar.php [hunk 1]
```diff
@@ -242,4 +242,59 @@ public function compileTruncate(Builder $query)
...
+     * @param  string  $value
+     * @return string
+     */
+    protected function wrapValue($value)
+    {
+        if ($value === '*') {
+            return $value;
+        }
+
+        if (str_contains($value, '->')) {
```
Comment (lucasmichot) [inline src/Illuminate/Database/Query/Grammars/PostgresGrammar.php]: `Str::contains`

Comment (themsaid) [inline src/Illuminate/Database/Query/Grammars/PostgresGrammar.php]: Makes sense. Thanks

File: src/Illuminate/Database/Query/Grammars/PostgresGrammar.php [hunk 2]
```diff
@@ -3,6 +3,7 @@
 namespace Illuminate\Database\Query\Grammars;
 
 use Illuminate\Database\Query\Builder;
+use Illuminate\Support\Str;
```
Comment (lucasmichot) [inline src/Illuminate/Database/Query/Grammars/PostgresGrammar.php]: Within this project, imports should be sorted by line-length, and then alphabetically.

Comment (themsaid) [inline src/Illuminate/Database/Query/Grammars/PostgresGrammar.php]: One more thank you :)

**Pull request merged**